profile/rwahs.xml

```diff
@@ -7593,24 +7593,110 @@
       <description>Museum Users</description>
       <actions>
       </actions>
+      <typeLevelAccessControl>
+        <permission table="ca_objects" type="artwork" access="edit"/>
+        <permission table="ca_objects" type="costume" access="edit"/>
+        <permission table="ca_objects" type="museumArtefact" access="edit"/>
+        <permission table="ca_entities" type="family" access="edit"/>
```

Comment:
As discussed offline, this possibly falls into the YAGNI category and is possibly making a management overhead. These _can_ be changed through the UI, but this is a small enough set of roles for it to be clear as to which to assign to each user.